src/docfx/lib/log/Error.cs

```diff
@@ -84,21 +96,21 @@ public override string ToString()
             var end_line = source?.EndLine ?? 0;
             var column = source?.Column ?? 0;
             var end_column = source?.EndColumn ?? 0;
-
-            return JsonUtility.Serialize(new
+            return JsonUtility.Serialize(new ErrorItem
```

Comment:
nit: instead of creating a class, how about
```suggestion
            return JsonUtility.Serialize(new JObject
```

Reply:
Is it possible to support `[JsonProperty("ms.author")]` within a JObject?

---
In reply to: [530181798](https://github.com/dotnet/docfx/pull/6812#discussion_r530181798) [](ancestors = 530181798)